Kitodo/src/main/java/org/kitodo/production/metadata/copier/DataCopier.java

```diff
@@ -42,13 +45,39 @@
      *             may be thrown if the program is syntactically wrong
      */
     public DataCopier(String program) throws ConfigurationException {
-        List<String> commands = Arrays.asList(program.split(";"));
+        List<List<String>> commands = parse(program);
         rules = new ArrayList<>(commands.size());
-        for (String command : commands) {
+        for (List<String> command : commands) {
             rules.add(DataCopyrule.createFor(command));
         }
     }
 
+    private static List<List<String>> parse(String input) {
```

Comment:
please give a more descriptive name. what are you parsing and what are you parsing it for?